turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksSwipeRefreshLayout.java

```diff
@@ -0,0 +1,28 @@
+package com.basecamp.turbolinks;
```

Comment:
Need Javadoc comments throughout, both for the class and each method. See the other classes for an example.